newm-server/src/main/kotlin/io/newm/server/features/playlist/PlaylistRoutes.kt

```diff
@@ -65,7 +65,11 @@ fun Routing.createPlaylistRoutes() {
                         }
                     }
                     get {
-                        call.respond(repository.getSongs(call.playlistId))
+                        with(call) {
+                            respond(
```

Comment:
Is the `createPlaylistRoutes` something we are paginating?

Reply:
No. Only the "GET" operations that return lists.